screen.go

```diff
@@ -52,10 +52,9 @@ func (s *screen) up(i string) {
 	s.y = int(math.Max(0, float64(s.y)))
 }
 
-// Move the cursor down, if we can
+// Move the cursor down
 func (s *screen) down(i string) {
 	s.y += pi(i)
-	s.y = int(math.Min(float64(s.y), float64(len(s.screen)-1)))
```

Comment:
Oh right! I actually did try that as a fix - but I was getting "Array out of bounds" type errors. I'll do some manual testing tomorrow morning and if it's all good - I'll 🚀 it to production. Thanks!!